apps/auction2/templates/index.jade

```diff
@@ -14,12 +14,18 @@ block body
       .auction2-commercial-filter
         .cf-headline
         .cf-sidebar
-          .cf-sidebar__filter.cf-sidebar__mediums
+          .cf-sidebar__filter.cf-sidebar__estimate-range
```

Comment:
This is so very minor but if you are going to do BEM the modifier would include the original class:

```
.cf-sidebar__filter.cf-sidebar__filter--estimate-range
```